src/chrome/content/rules/aastocks.com.xml

```diff
@@ -74,12 +75,14 @@
 -->
 <ruleset name="AASTOCKS.com (partial)">
 	<target host="www.aastocks.com" />
+	<test url="http://www.aastocks.com/tc/resources/Images/news/icon_video.png" />
 	<!-- 
 		On Firefox, enforcing HTTPS on the homepages will break the "Real-time
 		Latest Quote" functionalities because the Access-Control-Allow-Origin
 		header requiring HTTP. This bug does not affect Chromium browser (by 
 		default) or Firefox in HTTP Nowhere mode (#15606)
```

Comment:
I just had a closer look at this, but I think it's a mixed content issue.

Reply:
I have updated the comment accordingly in b94e3c7, thanks!!